lib/bolt/inventory.rb

```diff
@@ -60,9 +60,9 @@ def initialize(data, config = nil)
       @data = data ||= {}
       @groups = Group.new(data.merge('name' => 'all'))
       @group_lookup = {}
-      @target_vars = {}
-      @target_facts = {}
-      @target_features = {}
+      @target_vars = data['target_vars'] ||= {}
```

Comment:
Passing these in data now means someone can add them in an inventory file. We should pass them as a separate hash.